// @thirdweb-dev/react/evm/dist/thirdweb-dev-react-evm.cjs.dev.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var coinbaseWallet = require('../../dist/headlessConnectUI-314a447c.cjs.dev.js');
var useSafe = require('../../dist/useSafe-4bcb6ba3.cjs.dev.js');
var useMagic = require('../../dist/useMagic-d914d0f0.cjs.dev.js');
var useWalletConnect = require('../../dist/useWalletConnect-0991db83.cjs.dev.js');
var reactCore = require('@thirdweb-dev/react-core');
require('../../dist/basic-cf8743c1.cjs.dev.js');
require('@emotion/react');
require('@emotion/styled');
require('react/jsx-runtime');
require('@radix-ui/colors');
require('@thirdweb-dev/wallets');
require('detect-browser');
require('react');
require('react-qr-code');
require('@radix-ui/react-dialog');
require('@radix-ui/react-icons');
require('@tanstack/react-query');
require('@radix-ui/react-popover');
require('copy-to-clipboard');
require('@radix-ui/react-tooltip');
require('ethers');
require('@radix-ui/react-tabs');
require('fuse.js');
require('@radix-ui/react-dropdown-menu');
require('@thirdweb-dev/chains');
require('tiny-invariant');



exports.useIsWalletModalOpen = coinbaseWallet.useIsWalletModalOpen;
exports.useSetIsWalletModalOpen = coinbaseWallet.useSetIsWalletModalOpen;
exports.useSafe = useSafe.useSafe;
exports.useMagic = useMagic.useMagic;
exports.ConnectWallet = useWalletConnect.ConnectWallet;
exports.MediaRenderer = useWalletConnect.MediaRenderer;
exports.NetworkSelector = useWalletConnect.NetworkSelector;
exports.ThirdwebNftMedia = useWalletConnect.ThirdwebNftMedia;
exports.ThirdwebProvider = useWalletConnect.ThirdwebProvider;
exports.Web3Button = useWalletConnect.Web3Button;
exports.useCoinbaseWallet = useWalletConnect.useCoinbaseWallet;
exports.useFrameWallet = useWalletConnect.useFrameWallet;
exports.useInstalledWallets = useWalletConnect.useInstalledWallets;
exports.useMetamask = useWalletConnect.useMetamask;
exports.usePaperWallet = useWalletConnect.usePaperWallet;
exports.usePaperWalletUserEmail = useWalletConnect.usePaperWalletUserEmail;
exports.useRainbowWallet = useWalletConnect.useRainbowWallet;
exports.useResolvedMediaType = useWalletConnect.useResolvedMediaType;
exports.useTrustWallet = useWalletConnect.useTrustWallet;
exports.useWalletConnect = useWalletConnect.useWalletConnect;
exports.useWalletConnectV1 = useWalletConnect.useWalletConnectV1;
Object.keys(reactCore).forEach(function (k) {
  if (k !== 'default' && !exports.hasOwnProperty(k)) Object.defineProperty(exports, k, {
    enumerable: true,
    get: function () { return reactCore[k]; }
  });
});
